styles.css

```diff
@@ -1 +1,102 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+.container{
+    background-color: #FFBF69;
+    display: grid;
+    grid-gap: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px minmax(50px) 50px ;
+    grid-template-areas:
+    "grid-a"
+    "grid-b"
+    "grid-c"
+    "grid-d"
+    "grid-e"
+    "grid-f";
```

Comment:
je mist tussen grid-e en grid-e een lege 50px minmax(50px) lost dat niet op. Hier had je in de grid-template-area een . kunnen zetten: Dus

"grid-e"
"."
"grid-f"

een . is een leeg vlak in je grid.